util/dvsim/SimCfg.py

```diff
@@ -659,6 +660,14 @@ def create_bucket_report(buckets):
                     results_str += self.cov_report_deploy.cov_results
                     self.results_summary[
                         "Coverage"] = self.cov_report_deploy.cov_total
+
+                    # Copy cov_report to the report directory and create a
+                    # symlink to the latest directory.
+                    dest_target_dir = os.path.join(self.results_path, "cov_report")
```

Comment:
I thought this function replicated `src` at `dest`, i.e. it should already create a dir called `cov_report` at the dest.